app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class RentsController < ApiController
+      before_action :authenticate_user!
+
+      def index
+        rents = Rent.all
+        render_paginated rents, each_serializer: RentSerializer
+      end
+
+      def find_rents_by_user
+        rents = Rent.where(user_id: params[:user_id])
+        render_paginated rents, each_serializer: RentSerializer
+      end
+
+      def find_rents_by_book
+        rents = Rent.where(book_id: params[:book_id])
+        render_paginated rents, each_serializer: RentSerializer
+      end
```

Comment:
Estos no eran necesarios, pero igual aprovecho y te muestro algo.
fijate que hacer un action en tu controller para cada parametro que vas a usar para filtrar, llega a un punto en el que no es mantenible (tu controller va a quedar gigante lleno de actions que aplican filtros), te recomiendo muchisimo este post: https://www.justinweiss.com/articles/search-and-filter-rails-models-without-bloating-your-controller/, en el describen de manera detallada la manera en la que general mente hacemos esto.